packages/backend/src/apps/lettersg/auth/verify-credentials.ts

```diff
@@ -0,0 +1,35 @@
+import type { IGlobalVariable } from '@plumber/types'
+
+import { getEnvironmentFromApiKey, LetterSgEnvironment } from '../common/api'
+
+import { verifyApiKey } from './verify-api-key'
+
+export default async function verifyCredentials(
+  $: IGlobalVariable,
+): Promise<void> {
+  const authData = $.auth.data
+
+  if (!authData || !authData.apiKey) {
+    throw new Error('Missing LetterSG API key')
+  }
+
+  const env = getEnvironmentFromApiKey($.auth.data?.apiKey as string)
+  let labelSuffix: string | null = null
+  switch (env) {
+    case LetterSgEnvironment.Prod:
+      labelSuffix = ' [PROD]'
+      break
+    case LetterSgEnvironment.Staging:
+      labelSuffix = ' [STAGING]'
+      break
+  }
```

Comment:
To remove label suffixes for actual production release

Reply:
done!